src/plugins/crypto/testmod_crypto.c

```diff
@@ -213,7 +213,7 @@ static void test_enc_and_dec_with_null()
 {
 	elektraCryptoHandle *handle;
 	KeySet *config;
-	Key *errorKey = keyNew(KEY_END);
+	Key *errorKey = keyNew ("", KEY_END);
```

Comment:
Yes, the space is ok! Actually, there also should be a space around the `*`. Kurt is writting a style checker to formalize it. But it won't be too strict.